Sum of digits of a number.py

```diff
@@ -1,7 +1,31 @@
-q=0
-n=int(input("Enter Number: "))
-while(n>0):
- r=n%10
- q=q+r
- n=n//10
-print("Sum of digits is: "+str(q))
+import sys
+
+
+def get_integer():
+    for i in range(3,0,-1):
+        num = input("enter a number:")
+        if num.isnumeric():
+            num = int(num)
+            return num
+        else:
+            print("enter integer only")
+            print(f'{i-1} chances are left')
```

Comment:
Use {chance if i <=1 else chances} use something like this, to have a proper string literal output for "chances" and "chance."

Reply:
For sure. Thank u @NitkarshChourasia